datahub-web-react/src/app/search/filters/AdvancedFilters.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FacetFilterInput, FacetMetadata } from '../../../types.generated';
+import { useUserContext } from '../../context/useUserContext';
+import { ANTD_GRAY } from '../../entity/shared/constants';
+import { AdvancedFilterSelectValueModal } from '../AdvancedFilterSelectValueModal';
+import { AdvancedSearchAddFilterSelect } from '../AdvancedSearchAddFilterSelect';
+import { AdvancedSearchFilter } from '../AdvancedSearchFilter';
+import { AdvancedSearchFilterOverallUnionTypeSelect } from '../AdvancedSearchFilterOverallUnionTypeSelect';
+import useAdvancedSearchSelectFilters from '../useAdvancedSearchSelectFilters';
+import { UnionType } from '../utils/constants';
+import { FilterButtonsWrapper, FlexSpacer, FlexWrapper } from './BasicFilters';
+import SaveViewButton from './SaveViewButton';
+import { TextButton } from './styledComponents';
+import { hasAdvancedFilters } from '../utils/hasAdvancedFilters';
+
+const AnyAllToggle = styled.div`
+    font-weight: 700;
+    margin-bottom: 8px;
+`;
+
+const EmptyStateSection = styled.div`
+    padding: 6px 20px;
+    background-color: ${ANTD_GRAY[2]};
+    border-radius: 5px;
+`;
+
+interface Props {
+    availableFilters: FacetMetadata[];
+    activeFilters: FacetFilterInput[];
+    unionType: UnionType;
+    onChangeFilters: (newFilters: FacetFilterInput[]) => void;
+    onChangeUnionType: (unionType: UnionType) => void;
+    showBasicFilters: () => void;
+}
+
+export default function AdvancedFilters({
```

Comment:
So here we're basiclly leaving alone the existing advanced filters experience and adding someting net new?

Reply:
kind of.. it's a new entrypoint and layout, but the underlying components are the same. so the picker for choosing a filter is the same and the component for displaying existing advanced filters is the same but has some styling changes to work with the new layout.

basically it's the same advanced search functionality as before, but just working with the new layout